dist/stat_data.py

```diff
@@ -952,6 +953,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_blocked_no_ts_checkpoint_race_2', 'eviction gave up due to detecting a tombstone without a timestamp ahead of the selected on disk update'),
     CacheStat('cache_eviction_blocked_no_ts_checkpoint_race_3', 'eviction gave up due to detecting a tombstone without a timestamp ahead of the selected on disk update after validating the update chain'),
     CacheStat('cache_eviction_blocked_no_ts_checkpoint_race_4', 'eviction gave up due to detecting update chain entries without timestamps after the selected on disk update'),
+    CacheStat('cache_eviction_blocked_no_ts_checkpoint_race_5', 'eviction gave up due to no progress being made'),
```

Comment:
We should remove the name of no ts as this one is not caused by an update wihtout a timestamp. I would name it cache_eviction_blocked_no_progress